pkg/component/monitoring/alertmanager/component.go

```diff
@@ -54,14 +64,30 @@ type Values struct {
 	Replicas int32
 	// AlertingSMTPSecret is the alerting SMTP secret.
 	AlertingSMTPSecret *corev1.Secret
+	// IngressValues contains configuration for exposing this AlertManager instance via an Ingress resource.
+	Ingress *IngressValues
 
 	// DataMigration is a struct for migrating data from existing disks.
 	// TODO(rfranzke): Remove this as soon as the PV migration code is removed.
 	DataMigration monitoring.DataMigration
 }
 
+// IngressValues contains configuration for exposing this AlertManager instance via an Ingress resource.
+type IngressValues struct {
+	// AuthSecretName is the name of the auth secret.
+	AuthSecretName string
+	// Host is the hostname under which the AlertManager instance should be exposed.
+	Host string
+	// SecretsManager is the secrets manager used for generating the TLS certificate if no wildcard certificate is
+	// provided.
+	SecretsManager secretsmanager.Interface
+	// WildcardCertName is name of wildcard TLS certificate which is issued for the ingress domain. If not provided, a
+	// self-signed server certificate will be created.
+	WildcardCertName *string
```

Comment:
This is slightly confusing to me as it seems to be the name of the secret, which contains the certificate, and not the name of the certificate.
WDYT?